packages/core/admin/admin/src/content-manager/components/BlocksInput/BlocksContent.tsx

```diff
@@ -206,34 +247,37 @@ const DragAndDropElement = ({ children, index, canDrag, canDrop }: DragAndDropEl
           paddingLeft={2}
           onDragStart={(event) => {
             const target = event.target as HTMLElement;
-            target.style.opacity = '0.5';
+            const currentTarget = event.currentTarget as HTMLElement;
+            if (target.getAttribute('role') !== 'button') {
+              event.preventDefault();
+            } else currentTarget.style.opacity = '0.5';
```

Comment:
Setting the style like this doesn't feel like the react way to me.

Could you try setting some state here (e.g. setIsDragging) and in onDragEnd, then setting the style based on that state?

Reply:
as explained here , https://github.com/strapi/strapi/pull/18852#discussion_r1413608657